pkg/server/status.go

```diff
@@ -17,14 +17,22 @@ limitations under the License.
 package server
 
 import (
-	"errors"
-
 	"golang.org/x/net/context"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
 // Status returns the status of the runtime.
 func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequest) (*runtime.StatusResponse, error) {
-	return nil, errors.New("not implemented")
+	runtimeReady := &runtime.RuntimeCondition{
+		Type:   runtime.RuntimeReady,
+		Status: true,
+	}
+	networkReady := &runtime.RuntimeCondition{
+		Type:   runtime.NetworkReady,
+		Status: true,
```

Comment:
Add a TODO to check whether the plugins are ready or they exist? I didn't review the networking code, so this may not make sense.